python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -88,10 +86,10 @@ def test_mosaic_cifar10(ray_start_4_cpus):
     result = train_mosaic_cifar10().metrics_dataframe
```

Comment:
Nit: See comment above.

```suggestion
    result = train_mosaic_cifar10(max_duration="5ep").metrics_dataframe
```

This'd make the test easier to understand.